api/datastore/init.go

```diff
@@ -1,15 +1,14 @@
-package bolt
+package datastore
 
 import (
 	"github.com/gofrs/uuid"
 	portainer "github.com/portainer/portainer/api"
-	"github.com/portainer/portainer/api/bolt/errors"
 )
 
 // Init creates the default data set.
 func (store *Store) Init() error {
 	instanceID, err := store.VersionService.InstanceID()
-	if err == errors.ErrObjectNotFound {
+	if store.IsErrObjectNotFound(err) {
```

Comment:
Why do it this way?   I don't see what was wrong with what we had before.

Reply:
its a dependency inversion - before we had a dependency on an internal to bolt error object, this way, we have an interface (this is a very common meme in the go stdlib for this reason)